Allocator.cpp

```diff
@@ -50,6 +56,13 @@ GC_API void GC_CALL GC_remove_event_callback(GC_on_event_proc fn, void* data)
     }
 }
 
+GC_API void GC_CALL GC_remove_all_event_callbacks()
+{
+    if (!!g_eventCallbacks) {
+        delete g_eventCallbacks;
```

Comment:
IMO this is wrong
how about set this variable also as null?
and we can check whatever the value is null or not in gcEventCallbacks()
we can also remove inited variable from the function.

Reply:
Thanks for the review.
I updated the patch so that event callback array is located inside `gcEventCallbacks()` function.
Every contact to this array should invoke `gcEventCallbacks` function including delete operation.